backend/src/test/java/com/redhat/cloud/notifications/routers/sources/SecretUtilsTestTestProfile.java

```diff
@@ -0,0 +1,20 @@
+package com.redhat.cloud.notifications.routers.sources;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+import java.util.Map;
+
+public class SecretUtilsTestTestProfile implements QuarkusTestProfile {
```

Comment:
There's a drawback with Quarkus test profiles: they trigger one (sometimes two) full Quarkus restart (including the whole Flyway migration) during the tests execution, which leads to longer tests and more resources consumed.

The `sources.psk` config value could be set with a simple `System.setProperty("sources.psk", SOURCES_PSK_TEST);` at the beginning of the tests execution. Quarkus should be happy with that.

Reply:
All right, I'm switching to that...